src/playground/index.jsx

```diff
@@ -27,7 +28,7 @@ if (supportedBrowser()) {
 
 } else {
     BrowserModalComponent.setAppElement(appTarget);
-    const WrappedBrowserModalComponent = AppStateHOC(BrowserModalComponent, true /* localesOnly */);
+    const WrappedBrowserModalComponent = AppStateHOC(TitledHOC(BrowserModalComponent, true /* localesOnly */));
```

Comment:
You shouldn't wrap this one, since it's just the unsupported browser modal. There's also an issue with parenthesis placement (`true` should be passed to AppStateHOC, not TitledHOC), which shows that we should change the API for AppStateHOC to be used like `AppStateHOC(true)(Component))` rather than the kind of odd and hard to read `AppStateHOC(Component, true)`.

Reply:
Sorry, the parenthesis was sloppy of me! Should have caught that.

Reply:
Done.